proto/cosmos/authz/v1beta1/query.proto

```diff
@@ -16,10 +16,17 @@ service Query {
     option (google.api.http).get = "/cosmos/authz/v1beta1/grants";
   }
 
-  // GranterGrants returns list of `Authorization`, granted by granter.
+  // GranterGrants returns list of `GrantAuthorization`, granted by granter.
   rpc GranterGrants(QueryGranterGrantsRequest) returns (QueryGranterGrantsResponse) {
     option (google.api.http).get = "/cosmos/authz/v1beta1/grants/{granter}";
```

Comment:
since we're free to change everything here, should we change it to https://github.com/cosmos/cosmos-sdk/pull/10944#discussion_r788778948?

I liked your 1st `grants/granter/...` `grants/grantee/...` proposal